ui/desktop/tests/acceptance/projects/settings/index-test.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module } from 'qunit';
+
+module('Acceptance | projects | settings | index', function () {});
```

Comment:
Maybe we should have the first test be you can navigate to it

Reply:
added!